cloudify_agent/shell/commands/configure.py

```diff
@@ -73,53 +77,100 @@ def _disable_requiretty():
 
 
 def _fix_virtualenv():
-
     """
     This method is used for auto-configuration of the virtualenv.
     It is needed in case the environment was created using different paths
     than the one that is used at runtime.
 
     """
+    # this code is mostly taken from virtualenv; we can't use virtualenv's
+    # make_environment_relocatable because it checks if the old shebang
+    # points to the virtualenv dir, and packaged agents use a non-existent
+    # /tmp path. So, we copied _fixup_scripts mostly intact, but disabled
+    # the shebang checking.
+    _make_environment_relocatable(VIRTUALENV)
+
+
+def _make_environment_relocatable(home_dir):
+    home_dir, lib_dir, inc_dir, bin_dir = path_locations(home_dir)
+
+    _fixup_scripts(bin_dir)
+    fixup_pth_and_egg_link(home_dir)
+
 
+def _fixup_scripts(bin_dir):
+    """Make scripts in bin_dir relative by rewriting their shebangs
+
+    Examine each file in bin_dir - if it looks like a python script, and has a
+    shebang - replace it with a new, "relative" shebang. (like
+    `virtualenv --relocatable` would)
+
+    The relative shebang is platform-specific, but on linux it will consist
+    of a /usr/bin/env shebang, and a python snippet that runs the `activate`
+    script.
+    """
     from cloudify_agent.shell.main import get_logger
     logger = get_logger()
 
-    bin_dir = '{0}/bin'.format(VIRTUALENV)
+    new_shebang = _get_relative_shebang()
+    for filename in _find_scripts_to_fix(bin_dir):
+        logger.info('Making script %s relative' % filename)
```

Comment:
if we already went the extra mile of copying text from virtualenv, lets modify the string formatting code so that is uses '{0} some test'.format(some_arg) so that is it more consistent with how we generally do string formatting